### PR TITLE
12.13.0

### PR DESCRIPTION
**Pull request by bstevensmetova**

### CODE REVIEW

File: ios/Tealeaf/TLFResources.bundle/TealeafBasicConfig.plist [hunk 1]
```diff
@@ -13,15 +13,15 @@
 	<key>JavaScriptInjectionDelay</key>
 	<integer>100</integer>
 	<key>KillSwitchEnabled</key>
-	<false/>
+	<true/>
```
Comment (metova-joe) [inline ios/Tealeaf/TLFResources.bundle/TealeafBasicConfig.plist:16]: was this intentional?

File: ios/Tealeaf/TLFResources.bundle/TealeafBasicConfig.plist [hunk 2]
```diff
@@ -13,15 +13,15 @@
...
 	<key>KillSwitchMaxNumberOfTries</key>
 	<integer>3</integer>
 	<key>KillSwitchTimeInterval</key>
 	<integer>180</integer>
 	<key>KillSwitchTimeout</key>
 	<integer>5</integer>
 	<key>KillSwitchUrl</key>
-	<string></string>
+	<string>https://lib-us-2.brilliantcollector.com/collector/switch/b6c3709b7a4c479bb4b5a9fb8fec324c</string>
```
Comment (metova-joe) [inline ios/Tealeaf/TLFResources.bundle/TealeafBasicConfig.plist:24]: we don't appear to have traditionally shipped this, is this something we should be including?

**Review by metova-joe — COMMENTED** [review]
Just a few comments regarding config values we may or may not want to be sending, however I know these could be removed on release, but left in to expedite in-house developer setup.

File: ios/Tealeaf/TLFResources.bundle/TealeafBasicConfig.plist (hunk 1 above)
Comment (bstevensmetova) [reply ios/Tealeaf/TLFResources.bundle/TealeafBasicConfig.plist:16]: Yes the TLFResources.bundle was included in the release zip I got from the TeaLeaf repo, no mods from me just plopped it in

Comment (ohernandezIBM) [reply ios/Tealeaf/TLFResources.bundle/TealeafBasicConfig.plist:16]: Currently killswitch is turned off by default.

File: ios/Tealeaf/TLFResources.bundle/TealeafBasicConfig.plist (hunk 2 above)
Comment (ohernandezIBM) [reply ios/Tealeaf/TLFResources.bundle/TealeafBasicConfig.plist:24]: If killswitch is enabled, then will need to add a base killswitch url.

**Review by ohernandezIBM — COMMENTED** [review]
Good for now, will need to review with Rob on killswitch defaults.

**Review by cyangibm — COMMENTED** [review]
Agree, should promote the version to master and latest release.

**Pull request merged**